index.d.ts

```diff
@@ -0,0 +1,52 @@
+declare module 'mnemonist' {
+    type HashFunc = ((item: any) => any)
+    type HashFuncTuple = [HashFunc, HashFunc];
+    class FuzzyMultiMap {
+        constructor(hashFuncs: HashFunc | HashFuncTuple, container?: any)
+        clear(): void;
+        add(item: any): FuzzyMultiMap;
+        set(key: any, item: any): FuzzyMultiMap;
+        get(key: any): any;        
+        has(item: any): boolean;
+        forEach(callback: (item: any) => any, scope: any): void;
+        values(): Iterator<any>;
+        inspect(): any;
+        [key: string]: any;
```

Comment:
What does this part mean?

Reply:
I was trying to put something (hopefully correct) for everything I saw in your JavaScript.
> [key: string]: any;
is an [indexable type](https://www.typescriptlang.org/docs/handbook/interfaces.html#indexable-types).  I'd say I'm not that confident in those last few and we could leave them off for now and focus on the primary methods.

Reply:
I spent a couple of hours trying various combinations of tsconfig.json, command line, npm script, command line options, etc. but didn't have any luck.  I'm stopping for the moment.  btw though it is very common to see all the types in a single file.  Just not sure if they were generated from individual or not.